# A list of detected bodies
astra_ros/Body[] bodies

# True if the floor_plane was found, false otherwise
bool floor_detected

# The floor defined as a plane relative to the camera's frame
Plane floor_plane